src/OrleansTestingHost/DefaultSiloBuilderFactory.cs

```diff
@@ -8,7 +12,10 @@ public sealed class DefaultSiloBuilderFactory : ISiloBuilderFactory
         public ISiloBuilder CreateSiloBuilder(string siloName, ClusterConfiguration clusterConfiguration)
         {
             ISiloBuilder builder = new SiloBuilder();
-            return builder.ConfigureSiloName(siloName).UseConfiguration(clusterConfiguration);
+            return builder.ConfigureSiloName(siloName)
+                .UseConfiguration(clusterConfiguration)
+                .ConfigureLogging(loggingBuilder => TestingUtils.ConfigureDefaultLoggingBuilder(loggingBuilder,
```

Comment:
There are tests that don't use the default silo builder factory, but instead provide their own.  Can you search for all impls of ISiloBuilderFactory and ensure they all configure the logging.